std/random.d

```diff
@@ -415,8 +415,8 @@ $(D x0).
         return _x;
     }
 
-///
-    @property typeof(this) save() @safe pure nothrow
```

Comment:
Why are you removing `@safe`, `pure`, and `nothrow`? In none of the cases that you've changed here does whether the function can be `@safe`, `pure`, or `nothrow` depend on template arguments. So, there's no reason to remove them and every reason to have them.